syft/frameworks/crypten/__init__.py

```diff
@@ -3,10 +3,38 @@
 import crypten.communicator as comm
 import crypten
 
+from syft.workers.base import BaseWorker
+
+
+RANK_TO_WORKER_ID = {}
+CID = None
+
+
+def get_worker_from_rank(cid: int, rank: int) -> BaseWorker:
+    """Find the worker running CrypTen party with specific rank in a certain computation.
+
+    Args:
+        cid: CrypTen computation id.
+        rank: rank of the CrypTen party.
+
+    Returns:
+        BaseWorker corresponding to cid and rank.
+    """
+    rank_to_worker_id = RANK_TO_WORKER_ID.get(cid, None)
+    if rank_to_worker_id is None:
+        raise RuntimeError(
+            "CrypTen computation not initiated properly, computation_id doesn't match any rank to"
+            "worker_id transaltion table"
+        )
+    return syft.local_worker._get_worker_based_on_id(rank_to_worker_id[rank])
```

Comment:
Are all workers recognized by the local_worker?

Reply:
From what I know, the local worker is automatically created when we import ```syft``` and it should know about all the workers. (@LaRiffle this is the case, right?)

Reply:
Yup! For virtual workers this is true. For GridNodes it knows automatically the client nodes